doc/v2/documentation_data_format.rst

```diff
@@ -761,23 +763,24 @@ Detailed field description
 - ``lowerBound`` [NUMERIC]
 
   Lower bound of the parameter used for estimation.
-  Optional, if ``estimate==0``.
+  Optional, if ``estimate==false``.
 
 - ``upperBound`` [NUMERIC]
 
   Upper bound of the parameter used for estimation.
-  Optional, if ``estimate==0``.
+  Optional, if ``estimate==false``.
 
 - ``nominalValue`` [NUMERIC]
 
   Some parameter value to be used if
   the parameter is not subject to estimation (see ``estimate`` below).
-  Optional, unless ``estimate==0``.
+  Optional, unless ``estimate==false``.
 
-- ``estimate`` [BOOL 0|1]
+- ``estimate`` ['true' or 'false']
```

Comment:
true|false in the table above, but fine for me

Reply:
Fixed.